docker/README.md

```diff
@@ -17,7 +16,14 @@ Check that `kvm` is enabled and writable by the user running the docker containe
 To build `recipe.yaml`:
 ```
 cd <PATH_TO_RECIPE_DIR>
-docker run --rm --interactive --tty --device /dev/kvm --user $(id -u) --workdir /recipes --mount "type=bind,source=$(pwd),destination=/recipes" --security-opt label=disable godebos/debos <RECIPE.yaml>
+docker run --rm --interactive --tty \
+  --device /dev/kvm \
+  --user $(id -u):$(id -g) \
+  --group-add $(getent group kvm | cut -d: -f3) \
```

Comment:
kvm has uaccess tagged on modern systems so this is also not needed

Reply:
right, just having a look at this again since i don't fully understand the permissions here.
kvm has uaccess tagged on my system, and my user is part of the kvm group:

```
$ ls -la /dev/kvm
crw-rw----+ 1 root kvm 10, 232 Jan 28 09:36 /dev/kvm

$ getfacl /dev/kvm
getfacl: Removing leading '/' from absolute path names
# file: dev/kvm
# owner: root
# group: kvm
user::rw-
user:obbardc:rw-
group::rw-
mask::rw-
other::---
```

debos with kvm backend runs when i run it locally outside of docker, but not inside a container it breaks without this hunk of the patch:
```
$ docker run --rm --interactive --tty --device /dev/kvm --user $(id -u) --workdir /recipes --mount "type=bind,source=$(pwd),destination=/recipes" --security-opt label=disable godebos/debos:bullseye doc/examples/example.yaml
Could not access KVM kernel module: Permission denied
qemu-system-x86_64: failed to initialize kvm: Permission denied
error starting kvm backend: <nil>
```

i am probably doing something wrong here, or my system is configured incorrectly!

Reply:
hm seems like if i remove the `--user` arg from docker then it works fine, but then i loose the file permissions on the recipies bind mount